tests/Core/InfoButton/FluentInfoButtonTests.razor

```diff
@@ -0,0 +1,23 @@
+﻿@using Xunit;
+@inherits TestContext
+@code
+{
+    public FluentInfoButtonTests()
+    {
+        JSInterop.Mode = JSRuntimeMode.Loose;
+        Services.AddSingleton(new LibraryConfiguration());
+    }
+
+    [Fact]
+    public void FluentInfoButton_Default()
```

Comment:
Can you add other Unit Test to validate all properties (one test by property)?

Reply:
This has not been resolved